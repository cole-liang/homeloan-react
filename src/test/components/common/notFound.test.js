import React from "react";
import { shallow } from "enzyme";
import NotFound from "../../../components/common/notFound";

describe("NotFound Component Test", () => {
  it("should render NotFound component", () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
