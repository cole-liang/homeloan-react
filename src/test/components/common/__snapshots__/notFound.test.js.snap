// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Component Test should render NotFound component 1`] = `
<styled.div>
  <Bootstrap(Row)
    className="noPadding notFoundBg flex-grow-1 justify-content-center align-items-center"
    noGutters={true}
  >
    <div>
      <div
        className="notFoundText"
      >
        404
      </div>
      <div
        className="notFoundDesc mb-5 px-2"
      >
        Oops, your page is magically gone, please try again.
      </div>
    </div>
  </Bootstrap(Row)>
</styled.div>
`;
